### PR TITLE
[Docs] Add EN get_started

### PR DESCRIPTION
**Pull request by tianleiSHI**
getstartEN

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started/15_minutes_object_detection.md [hunk 1]
```diff
@@ -1,3 +1,523 @@
...
+</div>
+
+Take the small dataset of cat as an example, you can easily learn MMYOLO object detection in 15 minutes. The whole process consists of the following steps:
+
+- [Installation](#installation)
+- [Dataset](#dataset)
+- [Config](#config)
+- [Training](#training)
+- [Testing](#testing)
+- [EasyDeploy](#easydeploy)
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:16]:
```suggestion
- [EasyDeploy](#easydeploy-deployment)
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 2]
```diff
@@ -1,3 +1,523 @@
...
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/17425982/220072078-48b88a08-6179-483e-b8d3-0549e1b465de.png" alt="image"/>
+</div>
+
+The cat dataset is located in the mmyolo project dir, and `data/cat/annotations` stores annotations in COCO format, and `data/cat/images` stores all images
+
+## Config
+
+Taking YOLOv5 algorithm as an example, considering the limited GPU memory of users, we need to modify some default training parameters to make them run smoothly. The core parameters to be modified are as follows:
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:77]:
```suggestion
Taking YOLOv5 algorithm as an example, considering the limited GPU memory of users, we need to modify some default training parameters to make them run smoothly. The key parameters to be modified are as follows:
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 3]
```diff
@@ -1,3 +1,523 @@
...
+
+- YOLOv5 is an Anchor-Based algorithm, and different datasets need to adaptively calculate suitable anchors
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:79]:
```suggestion
- YOLOv5 is an Anchor-Based algorithm, and different datasets need to calculate suitable anchors adaptively
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 4]
```diff
@@ -1,3 +1,523 @@
...
+- The default config is 8 cards, each card batch size is 16, now change it to a single card, each card batch size is 12
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:80]:
```suggestion
- The default config uses 8 GPUs with a batch size of 16 per GPU. Now change it to a single GPU with a batch size of 12.
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 5]
```diff
@@ -1,3 +1,523 @@
...
+- The default training epoch is 300. Change it to 40 epoch
+- Due to the dataset is too small, we choose fixed backbone weight
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:82]:
```suggestion
- Given the small size of the dataset, we opted to use fixed backbone weights
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 6]
```diff
@@ -1,3 +1,523 @@
...
+- In principle, after batch size is changed, the learning rate also needs to be linearly scaled, but the actual measurement found that it is not necessary
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:83]:
```suggestion
- In principle, the learning rate should be linearly scaled accordingly when the batch size is changed, but actual measurements have found that this is not necessary
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 7]
```diff
@@ -1,3 +1,523 @@
...
+
+# Adaptive anchor based on tools/analysis_tools/optimize_anchors.py
+anchors = [
+    [(68, 69), (154, 91), (143, 162)],  # P3/8
+    [(242, 160), (189, 287), (391, 207)],  # P4/16
+    [(353, 337), (539, 341), (443, 432)]  # P5/32
+]
+# Mix training 40 epoch
+max_epochs = 40
+# bs 为 12
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:107]:
```suggestion
# bs = 12
```

File: docs/en/get_started/15_minutes_object_detection.md [hunk 8]
```diff
@@ -1,3 +1,523 @@
...
+# The palette length must be greater than or equal to the length of the classes
+metainfo = dict(classes=class_name, palette=[(20, 220, 60)])
+
+# Adaptive anchor based on tools/analysis_tools/optimize_anchors.py
+anchors = [
+    [(68, 69), (154, 91), (143, 162)],  # P3/8
+    [(242, 160), (189, 287), (391, 207)],  # P4/16
+    [(353, 337), (539, 341), (443, 432)]  # P5/32
+]
+# Mix training 40 epoch
```
Comment (RangeKing) [inline docs/en/get_started/15_minutes_object_detection.md:105]:
```suggestion
# Max training 40 epoch
```

**Review by RangeKing — APPROVED** [review]
I reviewed the first two sections and gave some detail comments. Please recheck the full text.

**Comment by tianleiSHI** [comment]
> 

Thanks a lot, I'll check it again.

**Pull request closed without merging**